lib/modules/manager/pre-commit/extract.ts

```diff
@@ -34,7 +34,10 @@ function determineDatasource(
   }
   if (hostname === 'gitlab.com' || detectPlatform(repository) === 'gitlab') {
     logger.debug({ repository, hostname }, 'Found gitlab dependency');
-    return { datasource: GitlabTagsDatasource.id };
+    return {
+      datasource: GitlabTagsDatasource.id,
+      registryUrls: ['https://' + hostname],
```

Comment:
only set registry url If it's not gitlab.com

Reply:
Hi  @viceice 
Thanks for the review. I implemented the changes and added some tests. Please have a look.